### PR TITLE
Implement delayed actions

### PR DESCRIPTION
**Pull request by Peque**
A self-contained change that could be added to master while we keep making other changes.

### CODE REVIEW

**Comment by Flood1993** [comment]
Very nice code. Feel free to merge it into master @Peque ! :+1:

File: osbrain/common.py [hunk 1]
```diff
@@ -77,6 +77,41 @@ def loop():
...
+    """
+    event = Event()
+
+    def wait():
+        if event.wait(delay):
+            return
+        action(*args)
+
+    Thread(target=wait).start()
+    event.stop = event.set
```
Comment (Flood1993) [inline osbrain/common.py]: I'm not sure what it does... Could you explain?

Comment (Peque) [inline osbrain/common.py]: That line simply allows you to use `timer.stop` instead of `timer.set`, which I think is more intuitive.

Comment (Flood1993) [inline osbrain/common.py]: Ah, I see :+1:

**Pull request merged**